data/phone/text/jack_caller.asm

```diff
@@ -177,7 +177,7 @@ UnknownText_0x1745c2:
 	cont "away and tell you."
 	done
 
-UnknownText_0x174638:
+JackReminderText:
```

Comment:
`JckRememberYourPromiseText`: more descriptive, and no reason to change "remember" into "reminder".

Reply:
"XReminderText" is used for almost every script out there. Same with "XLaterText". It's a generic "battle reminder" text.